examples/simple/roc.config.js

```diff
@@ -3,8 +3,9 @@ module.exports = {
         runtime: {
             applicationName: 'Jest Test',
         },
-        build: {
-            routes: 'src/routes.js',
+        test: {
+            jest:{
```

Comment:
We could add it, but to create a test that tests if jest produces junit reports is complicated an d not worth the time and effort.